README.md

```diff
@@ -0,0 +1,17 @@
+# Service Access Layer
+
+Provides tools to generate from XML description language and database bindings
+for LSST commands, events and telemetry. Depends on ts\_openslice.
+
+# Usage
+
+To get needed github projects and setup environment variables, do:
+
+```bash
+git clone https://github.com/lsst-ts/ts_opensplice
+git clone https://github.com/lsst-ts/ts_sal
+source ts_sal/setup.env
```

Comment:
Users will probably to edit setup.env first (certainly that is what the file itself says), or define some other env vars before sourcing setup.env.

Installing the dependencies for ts_sal and configuring it is a big deal. It would be *great* to document the details, but I fear it may be more information than reasonably belongs in the README.

Also I suspect the requirements vary depending on what you want ts_sal to do. I'm not sure if it needs OpenSplice to make IDL files for use by ts_salobj (though you need it to use those IDL files). It surely needs OpenSplice in order to make C++, Java and LabVIEW libraries.

Reply:
Users don't have to edit ts_sal/setup.env, as it now provides sensible defaults. It works. No need for extra repos.